packages/js/src/components/WincherPerformanceReport.js

```diff
@@ -222,10 +312,10 @@ const WincherPerformanceReport = ( props ) => {
 				{ __( "Top performing keyphrases on your site", "wordpress-seo" ) }
 			</WincherSEOPerformanceReportHeader>
 
-			<GetUserMessage { ...props } />
+			<GetUserMessage { ...props } data={ data } />
 
-			{ isLoggedIn && data && ! isEmpty( data ) && ! isEmpty( data.results ) && <Fragment>
-				<TableExplanation />
+			{ data && ! isEmpty( data ) && ! isEmpty( data.results ) && <Fragment>
+				<TableExplanation isLoggedIn={ isLoggedIn } />
 
 				<WincherSEOPerformanceTableWrapper>
 					<table className="yoast yoast-table">
```

Comment:
I think we ought to disable pointer events and user select on the table (not wrapper) if it's blurred:

```
pointer-events: none;
user-select: none;
```